.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: Build, Test, Package and Push
+
+# Controls when the action will run. 
+on: 
+  # Triggers the workflow on pull request events and merges/pushes to main
+  push:
+    branches:
+      - main
+      - release/*
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```

Comment:
do we want to go with 20.04, 18.04 is kinda old these days,.

Reply:
`ubuntu-latest` would be nice (this is currently `20.04`)

Reply:
Sticking with `18.04` for the moment, a library OctoVersion uses for Git access is throwing an exception on `20.04`

Reply:
Fixed, we're on `20.04`